lib/launcher.rb

```diff
@@ -7,7 +7,7 @@
 require 'rack'
 require 'webrick'
 begin
-  require 'puma'
+  Bundler.require(:puma)
```

Comment:
Please test this on RPM installation, I think this will not work. We actually turn bundler off for RPM installations and use special stub called `bundler_ext` so it works seamlessly with Bundler for development.

Reply:
Will check it out at the end of my work (soon, very soon :) )